### PR TITLE
✨ Add loft to freecadapi and geometry

### PR DESCRIPTION
**Pull request by je-cook**
## Description

Add loft to geometry tools through the freecad api

## Checklist

I confirm that I have completed the following checks:

- [ ] Tests run locally and pass `pytest tests --reactor`
- [ ] Code quality checks run locally and pass `pre-commit run --from-ref develop --to-ref HEAD`
- [ ] Documentation built locally and checked `sphinx-build -W documentation/source documentation/build`

### CODE REVIEW

File: bluemira/geometry/tools.py [hunk 1]
```diff
@@ -983,6 +983,39 @@ def sweep_shape(
...
+    Loft between a set of profiles.
+
+    Parameters
+    ----------
+    profiles:
+        Profile(s) to loft between
+    solid:
+        Whether or not to create a Solid
+    ruled:
+        created a ruled shape
```
Comment (oliverfunk) [inline bluemira/geometry/tools.py:1003]:
```suggestion
        Create a ruled shape
```

Would also be useful to know briefly what a ruled shape is

File: bluemira/codes/_freecadapi.py [hunk 2]
```diff
@@ -1694,6 +1694,32 @@ def sweep_shape(
...
+    Loft between a set of profiles.
+
+    Parameters
+    ----------
+    profiles:
+        Profile(s) to loft between
+    solid:
+        Whether or not to create a Solid
+    ruled:
+        created a ruled shape
```
Comment (oliverfunk) [inline bluemira/codes/_freecadapi.py:1710]:
```suggestion
        Create a ruled shape
```

**Review by oliverfunk — COMMENTED** [review]
Great! Just one small comment

File: bluemira/geometry/tools.py (hunk 1 above)
Comment (je-cook) [reply bluemira/geometry/tools.py:1003]: I'll just put this in the docs, it'll be better than my description https://en.wikipedia.org/wiki/Ruled_surface

**Review by oliverfunk — APPROVED** [review]

**Pull request merged**